README.md

```diff
@@ -19,12 +19,7 @@ Watch videos together from the comfort of your own room!
 [![Figma](https://img.shields.io/badge/Design-Figma-1ABCFE.svg?logo=figma)](https://www.figma.com/file/2NjbwtE7o72YfgAG32m3DU/website?node-id=2%3A190)
 
 To start your Phoenix server:
-
-- Install dependencies with `mix deps.get`
-- Create and migrate your database with `mix ecto.setup`
-- Install Node.js dependencies with `npm install` inside the `assets` directory
-- Create a new file in the config folder called `ueberauth.secret.exs`, copy the contents of `ueberauth.secret.exs.example` into it, and fill in the information
-- Start Phoenix endpoint with `mix phx.server`
+Just run `docker-compose up`. If its not working, it is because you have an arm chip. You can resolve the issue by adding `arm` to the end of the [Dockerfile](Dockerfile) (so the version tag will be `1.12v2arm`). Make sure you don't commit this.
```

Comment:
Please change the language. The word just doesn't really sound professional. Also I'd change the "if it's not working" to " if you have an arm chip"